pkgm.ts

```diff
@@ -135,7 +145,13 @@ async function install(args: string[]) {
     runtime_env,
     ...to_install,
   ];
-  const cmd = needs_sudo ? "/usr/bin/sudo" : args.shift()!;
+  let cmd = ""
+  if (needs_sudo) {
+    cmd = "/usr/bin/sudo";
+    args.unshift("-E"); // we already cleared the env, it's safe
```

Comment:
Without `-E`, `sudo` will not honor the `PATH` passed with `env`. It just happened to work because `sudo`'s own `PATH` already contained the paths for pkgx.